### PR TITLE
Info text about user bids below buttons - NFT widget

### PR DESCRIPTION
**Pull request by eldiegod**
fixes: #2371

### CODE REVIEW

File: packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx [hunk 1]
```diff
@@ -126,14 +126,25 @@ export const NftWidget: React.FC<NftWidgetProps> = ({
...
     )
-    const WithdrawLastBit = ({ secondary }: { secondary?: boolean }) => (
-      <GridItem colSpan={buttonColumnSpan}>
-        <ButtonGrid data-size={size}>
-          <Button variant={secondary ? 'secondary' : undefined} fullWidth size={buttonSize}>
-            Withdraw last bid
-          </Button>
-        </ButtonGrid>
-      </GridItem>
+    const WithdrawLastBid = ({ secondary }: { secondary?: boolean }) =>
```
Comment (drillprop) [inline packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx]: Can we name this differently? WithdrawBidFromPreviousAuction maybe?

File: packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx [hunk 2]
```diff
@@ -126,14 +126,25 @@ export const NftWidget: React.FC<NftWidgetProps> = ({
...
+      bidFromPreviousAuction ? (
+        <>
+          <GridItem colSpan={buttonColumnSpan}>
+            <Button variant={secondary ? 'secondary' : undefined} fullWidth size={buttonSize}>
+              Withdraw last bid
+            </Button>
+            <Text as="p" margin={{ top: 2 }} variant="t100" secondary align="center">
+              You bid {formatNumberShort(Number(bidFromPreviousAuction?.amount))} tJOY on{' '}
+              {formatDateTime(bidFromPreviousAuction.createdAt)}
```
Comment (drillprop) [inline packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx:139]: The app is breaking and I can't do anything when I go to the video with an open auction with my bid added. I'm getting: 
image
I assume that it comes from here.

**Review by drillprop — CHANGES_REQUESTED** [review]

**Comment by eldiegod** [comment]
Added suggestions, removed double settle buttons bug, and added bidlock info text

**Review by drillprop — APPROVED** [review]
I will need your changes in my "withdraw bid" PR, so let's merge this earlier. Looks good to me!

**Pull request merged**